product-service/handler.ts

```diff
@@ -2,6 +2,7 @@ import 'source-map-support/register';
 
 import getProductsList from './handlers/getProductsList';
 import getProductById from './handlers/getProductsById';
-import addProduct from './handlers/addProduct'
+import addProduct from './handlers/addProduct';
+import catalogBatchProcess from "./handlers/catalogBatchProcess";
 
-export { getProductsList, getProductById, addProduct };
+export { getProductsList, getProductById, addProduct, catalogBatchProcess };
```

Comment:
You could use the following syntax, e.g.:
`export {addProduct} from './handlers/addProduct';`